.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,5 +1,5 @@
 name: Bug report
-description: Report problems and issues
+description: Report problems and issues specific to this CORTX component. Please first check to make sure this has not yet been reported.
```

Since you don't like "project", I would still say something like:

```suggestion
description: Report problems and issues specific to this repository. Please first check to make sure this has not yet been reported.
```
I wouldn't call everything a "CORTX component". I don't see something like cortx-re cortx-k8s, cortx-test, etc. as "components".

---

How would you feel about "CORTX submodule"?

---

If I _had_ to choose between "submodule" and "component", then I guess I'd go with "component". 😆

---

@andriytk , any thoughts / preferences about this? The goal is to not be confusing and help people understand the difference between the overall CORTX project and the specific repo where they are submitting an Issue. Unfortunately, there is no way to have variable substitution in github issue templates. If we could have variable substitution, then it would be much easier.

Here is an open github feature request to add variable substitution into templates:
https://github.community/t/feature-request-variable-substitution-for-template-repositories/190517 
Please upvote it. :)